tests/acceptance/test_acceptance/test_activate.py

```diff
@@ -267,7 +266,7 @@ def test_activate_403(session_override_sdk_key):
     payload = '{"userId": "matjaz", "userAttributes": {"attr_1": "hola"}}'
     params = {"type": "experiment"}
 
-    request, request_result = create_and_validate_request(ENDPOINT_ACTIVATE, 'post', payload, params)
+    request, request_result = create_and_validate_request(ENDPOINT_ACTIVATE, 'post', payload=payload, params=params)
```

Comment:
why aren't you using **create_and_validate_request_and_response** in this test

Reply:
didn't use create_and_validate_request_and_response in any test cases with **session_override_sdk_key** arg because these have additional  **with pytest.raises(requests.exceptions.HTTPError):** check